tests/ExternRefTests.cs

```diff
@@ -49,7 +49,7 @@ public void ItHandlesNullReferences()
             var nullref = instance.GetFunction(Store, "nullref");
             inout.Should().NotBeNull();
 
-            (inout.Invoke(Store, null)).Should().BeNull();
+            (inout.Invoke(Store, ValueBox.AsBox((object?)null))).Should().BeNull();
```

Comment:
As these are tests, I think we can just omit the `?` here to remove the warning.

Reply:
This actually lead me to noticing that `AsBox` should accept `null` anyway.